rotkehlchen/tests/unit/test_tasks_manager.py

```diff
@@ -235,3 +239,59 @@ def test_check_premium_status(rotkehlchen_api_server):
             raise AssertionError(f'Premium check query was not completed within {timeout} seconds') from e  # noqa: E501
 
         assert rotki.premium is None
+
+
+def test_exchanges_filtering(database, exchange_manager, function_scope_messages_aggregator):
```

Comment:
Test looks correct but the task manager is definitely not the right location for this. Add a new `test_exchanges.py` under the `unit` directory. Seems we don't have one.